core/app/models/spree/shipping_rate.rb

```diff
@@ -1,46 +1,48 @@
 module Spree
   class ShippingRate < Spree::Base
+    attr_writer :order
```

Comment:
@mamhoff would https://github.com/solidusio/solidus/pull/965 be sufficient to remote the `attr_writer` related code here?